content/en/docs/Writing policies/tips.md

```diff
@@ -58,6 +58,8 @@ These are some tips and tricks you can use when putting together your Kyverno po
 
 * `validate` rules have no precedence/overriding behavior, so even though a rule may be written to either allow or deny a resource/action, one cannot counteract the other. For example, a rule written to ensure all images come from registry `reg.corp.com` and another rule written to ensure they do **not** come from `reg.corp.com` will effectively render all image pulls impossible and nothing will run. Where the rule is defined is irrelevant.
 
+* The choice between using a `pattern` statement or a `deny` statement depends largely on the data you need to consider. `pattern` works on incoming (new) objects while `deny` works on operations and existing objects.
```

Comment:
change: "The choice between using a `pattern` statement or a `deny` statement depends largely on the data you need to consider. `pattern` works on incoming (new) objects while `deny` works on operations and existing objects"

to:

"The choice between using a `pattern` statement or a `deny` statement depends largely on the data you need to consider; `pattern` works on incoming (new) objects while `deny` can additionally work on variable data such as the API operation (CREATE, UPDATE, ...), old object data, and ConfigMap data."

Reply:
Added https://github.com/kyverno/website/pull/54/commits/ca0c2c2a9e561a23006a15c60cecebe0998825c6 which should address